crates/context_aware_config/src/api/dimension/handlers.rs

```diff
@@ -131,3 +144,54 @@ async fn get(
 
     Ok(HttpResponse::Ok().json(dimensions_with_mandatory))
 }
+
+pub fn delete_dimension_helper(
```

Comment:
Few things:
- why is this function public
- why is it required? This code can be in the handler itself

Reply:
this function need not to be public , i have removed